src/components/editor/document-bar/document-info/document-info-line.tsx

```diff
@@ -4,12 +4,13 @@ import { IconName } from '../../../common/fork-awesome/types'
 
 export interface DocumentInfoLineProps {
   icon: IconName
+  isInline?: boolean
```

Comment:
IMHO "isInline" is a weird name for a variable that sets the icon size. How about just providing the icon size as optional argument? That would take a huge chunk of complexity out of this simple component.

Reply:
Good point. I initially also changed the font-size with this setting but as setting the font-size to inherit is enough to make it work in all places, I'm going to change that as you mentioned it here.

Reply:
Done in 4695bae